backend/tests/integration/message_bus/operations/request/test_proposed_change.py

```diff
@@ -193,29 +191,3 @@ async def test_run_generators_validate_requested_jobs(
         "request.proposed_change.refresh_artifacts",
         "request.proposed_change.repository_checks",
     ]
-
-
-async def test_cancel(
-    prepare_proposed_change: str,
-    db: InfrahubDatabase,
-    test_client: InfrahubTestClient,
-):
-    message = messages.RequestProposedChangeCancel(
-        proposed_change=prepare_proposed_change,
-    )
-    integration_helper = IntegrationHelper(db=db)
-    bus_pre_data_changes = BusRecorder()
-    admin_token = await integration_helper.create_token()
-    config = Config(api_token=admin_token, requester=test_client.async_request)
-    client = InfrahubClient(config=config)
-    fake_log = FakeLogger()
-    services.service._client = client
-    services.service.log = fake_log
-    services.service.message_bus = bus_pre_data_changes
-    services.prepare(service=services.service)
-    await cancel(message=message, service=services.service)
-    assert fake_log.info_logs == ["Canceling proposed change as the source branch was deleted"]
-    proposed_change: CoreProposedChange = await NodeManager.get_one_by_id_or_default_filter(
-        db=db, id=prepare_proposed_change, kind=InfrahubKind.PROPOSEDCHANGE
-    )
-    assert proposed_change.state.value.value == ProposedChangeState.CANCELED.value
```

Comment:
should there be new unit tests to cover the deleted ones in this module and those below?

Reply:
I'll add a functional test for it in a separate PRs